src/data_loader.py

```diff
@@ -0,0 +1,22 @@
+import tensorflow as tf
+from image_utils import data_generator
+
+def create_dataset() -> tf.data.Dataset: # TODO: Modify to accept other base data dirs
+    output_sig = (
+        tf.TensorSpec(shape=(16, 512, 512), dtype=tf.float32),  # Images
```

Comment:
Can we make this so that the images are in channels last format, i.e., (512, 512, 16) instead of (16, 512, 512)? The model is written in channels last format.